src/hooks/useArticleLinksNavigation.js

```diff
@@ -103,7 +103,7 @@ export const useArticleLinksNavigation = (
 
 const makeLinkClickEvent = link => {
   const title = link.getAttribute('title')
-  if (title && link.pathname !== '/') {
+  if (title && !['/', '/index.html'].includes(link.pathname)) {
```

Comment:
Could you explain what's gong on here, give examples oaf paths this code encounters and how it handles them?

Reply:
in the normal markup with having title attribute, we would like to trigger the article preview

```html
<a href="/wiki/Sean_Chiplock" title="Sean Chiplock">Sean Chiplock</a>
```

in the idea markup going to the specific anchor within the same article

```html
<a href="#Related_characters">See below</a>
```

but it's normal to see it has the title attribute, which conflicts with the first statement we have

```html
<a href="#Related_characters" title="C">See below</a>
```

therefore the need to the extra guard of the title attribute with the pathname

---

the perfect solution will be, passing all the anchor name to the hook and check against when having the hash in the link

```pseducode
hash => check all anchor, it matches then go to the anchor section
if doesn't match, continue the logic of starting with title
```

since we pass gallery items in this pr https://github.com/wikimedia/wikipedia-kaios/pull/239, are we fine also pass the sections list within the hook?